screens/ChooseStyleScreen.tsx

```diff
@@ -0,0 +1,41 @@
+import { useState } from 'react'
+import { StyleSheet, ImageBackground, Image, ScrollView, Dimensions } from 'react-native'
+import { Text, View } from 'components/Themed'
+import Button from 'components/Button'
+import ThemePicker from 'components/ThemePicker'
+import ProjectThemes, { IProjectTheme } from 'constants/ProjectThemes'
+
+const { width, height } = Dimensions.get('screen')
+const containerWidth = width * 0.8
+
+export default function ChooseStyleScreen({ route, navigation }) {
```

Comment:
could add types for the function parameters so it knows what to expect

Reply:
done